xxx-package-module-property.md

```diff
@@ -107,8 +140,11 @@ These rules are taken into account in the draft specification included below.
 ## 4.3 Loading Modules without a package.json
 
 If writing a `.js` file without any `package.json` configuration, it remains
-possible to opt-in to ES modules by indicating this by either using the `.mjs`
-file extension or `"use module"` directive, which always take preference.
+possible to opt-in to ES modules by indicating this by using the `.mjs`
+extension.
+
+When running NodeJS in a repl mode, a flag could possibly be considered to
+treat the passed source as an ES module.
```

Comment:
I'm not certain this is possible, imports and exports are computed statically before file execution.

Exports in particular is probably impossible though to little concern. Imports may be able to be hacked around somehow.

I don't think this is directly related to this proposal however and will almost certainly need to exist regardless of which normal detection proposal is chosen.